src/App.tsx

```diff
@@ -1,194 +1,149 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+// import { UserWarning } from './UserWarning';
```

Comment:
Remove comment. Try it write obvious code that explains all we need without comments)
Fix all cases